index.html

```diff
@@ -7,6 +7,7 @@
   <title>Build Triage Links for Bugzilla Component(s)</title>
   <link rel="stylesheet" type="text/css" href="jquery-ui-1.11.4/jquery-ui.min.css">
   <link rel="stylesheet" type="text/css" href="index.css">
+  <link rel="icon shortcut" href="Green_bug_and_broom.svg" type="image/svg">
```

Comment:
Firefox [seems](http://caniuse.com/#feat=link-icon-svg) to be the only major browser that allows SVG favicons. How much do we care for triage-center?